include/xtensor-io/xfile_array.hpp

```diff
@@ -123,10 +124,10 @@ namespace xt
         self_type& operator=(self_type&&) = default;
 
         template <class OE>
-        xfile_array_container(const xexpression<OE>& e);
+        xfile_array_container(const xexpression<OE>& e, xfile_mode mode=load, value_type init_value=value_type());
 
         template <class OE>
-        xfile_array_container(const xexpression<OE>& e, const std::string& path);
+        xfile_array_container(const xexpression<OE>& e, const std::string& path, xfile_mode mode=load, value_type init_value=value_type());
```

Comment:
Passing an `init_value` does not really make sense here since the expression will be initialized from `e`.